### ISSUE TITLE
Payee/Payer fields are not available

### ISSUE THREAD

**Issue by rrnovaesjr** [Eqonomize/Eqonomize#166]
Hi,

After editing some transactions via ledger, I couldn't see the option to input the "Payee" or "Payer" option anymore. The headers were initially hidden, but I could select them to show:

![image](https://github.com/Eqonomize/Eqonomize/assets/25465829/0affe4b5-9607-4c5b-8451-61fb4c077191)

As you can see, I am filling the payee under the "Comments" tab so I don't lose track of what's going on.

This is what I see when inserting a new expense (or income):

![image](https://github.com/Eqonomize/Eqonomize/assets/25465829/9c42f380-c9f9-44db-ab08-8626f6b621f8)

I've been using v1.5.3 for quite some time now and the only thing I noticed differently is that I used the Ledger to create/edit incomes or expenses.

Will you please help me out? :pray:

**Comment by hanna-kn** [comment]
Make sure that "Show payee and quantity" in the settings menu is selected.

**Comment by rrnovaesjr** [comment]
Yep, that was it. I feel a bit stupid myself, but thank you anyway :pray:
